Chrome

APMichael opened the pull request:
Merged Chrome-based (7Star, 360 Browser, Coowon, SlimJet, SuperBird) entries.

Added outdated Chrome-based (7Star, 360 Browser, Coowon, SlimJet, SuperBird) entries to "Removed entries.ini": #194


I didn’t touch the entry [Comodo Dragon *]:
- Some of the FileKeys differ too much.
- In older versions of winapp2.ini the FileKeys were merged already, but someone moved them out into one big single entry again.

MoscaDotTo commented on Winapp2.ini:
```diff
@@ -441,105 +325,131 @@ FileKey5=%ProgramFiles%\Google\Chrome\Application\*\Installer|*.7z
 [JumpListIconsRecentClosed *]
```
I think this entry should perhaps be merged into History *

MoscaDotTo replied:
Addtly, it looks like there's another path here called "JumpListIconsMostVisited" that can be similarly cleaned. Both are for the jumplist that opens when you right click chrome on the windows task bar.

MoscaDotTo commented on Winapp2.ini:
```diff
@@ -441,105 +325,131 @@ FileKey5=%ProgramFiles%\Google\Chrome\Application\*\Installer|*.7z
-FileKey1=%LocalAppData%\Google\Chrome\User Data\Default\JumpListIconsRecentClosed|*.*|RECURSE
+FileKey1=%LocalAppData%\7Star\7Star\User Data\*\JumpListIconsRecentClosed|*.*|RECURSE
+FileKey2=%LocalAppData%\Coowon\Coowon\User Data\*\JumpListIconsRecentClosed|*.*|RECURSE
+FileKey3=%LocalAppData%\Google\Chrome\User Data\*\JumpListIconsRecentClosed|*.*|RECURSE
```
\Chrome\ here should be \Chrome*\ for Chrome SxS

APMichael commented:
Thanks for your replies. I'm not an expert for Chrome. Therefore, I just merged and moved the separated Chrome-based entries, using the remaining entries as reference. I updated the pull request with your suggestion.

MoscaDotTo commented:
LGTM 👍
